src/helpers/trayManager.ts

```diff
@@ -45,7 +45,7 @@ export class TrayManager {
 
   public startIfEnabled(): void {
     if (this.enabled) {
-      this.tray = new Tray(this.iconPath);
+      this.tray = new Tray(this.iconPath, this.stringToUuid(this.iconPath));
```

Comment:
Can we just pass it something like "android-messages-desktop" (+ "-development" if we are in dev mode)?

Reply:
The second parameter must be UUID, we can generate 2 different UUID's for production/dev versions.

Reply:
```
guid String (optional) Windows - Assigns a GUID to the tray icon. If the executable is signed and the signature contains an organization in the subject line then the GUID is permanently associated with that signature. OS level settings like the position of the tray icon in the system tray will persist even if the path to the executable changes. If the executable is not code-signed then the GUID is permanently associated with the path to the executable. Changing the path to the executable will break the creation of the tray icon and a new GUID must be used. However, it is highly recommended to use the GUID parameter only in conjunction with code-signed executable. If an App defines multiple tray icons then each icon must use a separate GUID.
```

Official docs say that we should not be setting this at all because we are not signing the code.

Reply:
Since GUID is permanently linked to specific path of the executable, we cannot use static GUID, it must be generated based on path of the executable. When AMD is packed, on runtime it unpacks itself into temp directory, which changes each time AMD is packed (`yarn dist`).
Just tested it with static GUID once the path changed after second packing no tray icon shown at all.

Reply:
I think you are misunderstanding what I meant. 

> However, it is highly recommended to use the GUID parameter only in conjunction with code-signed executable

this is the important line. We are not signing the executable therefore it is ill advised for us to use this parameter.

Reply:
I understand your concern, but I believe the warning is based on the idea of using hard coded GUID. We are using dynamic GUID.